Docker/base/Dockerfile

```diff
@@ -1,6 +1,14 @@
 # Use the official ROS Noetic base image
 FROM ros:noetic-ros-base
 
+RUN apt-get update && apt-get install -y locales\
```

Comment:
only change I would make is put ultralytics + scikit in base image (if it doesnt cause problems with the actions) cuz it is more of a base feature than a sim-specific thing. Would also remove the need for a sim image and we could just have base or main or whatever, poolside/jetson, and actions. And also it enables us to have vision integration testing later on if we want

Reply:
was it that slow because of ultralytics? can we try pushing with ultralytics in base image and seeing how long it takes? cuz if its not a significant slow down then makes sense to have it in base image imo